include/SettingsManager.hpp

```diff
@@ -95,145 +97,182 @@ struct SettingsData
 
 class SettingManager
 {
+    // All functions with code here are implicitly inlined by the compiler.
   public:
     SettingManager();
 
-    int getConnectionPort();
-    void setConnectionPort(int port);
+    int getConnectionPort() { return mSettings->value("companion_port", 10045).toInt(); }
+    void setConnectionPort(const int &port) { mSettings->setValue("companion_port", port); }
```

Comment:
Don't use `const &` for basic types. In fact, compilers might optimize it to pass by value.

Reply:
It's not necessary, but without optimization, this might be slower than pass by value, because the value is smaller than a pointer.

Reply:
Well, in fact it's going to be inlined, so const & won't really make a difference here. The function is implicitly inlined.